cachito/workers/tasks/utils.py

```diff
@@ -233,6 +234,26 @@ def set_packages_and_deps_counts(request_id: int, packages_count: int, dependenc
     )
 
 
+def sort_packages_and_deps_in_place(packages):
```

Comment:
It would be good to have a unit test for this

Reply:
Done!